Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Runs on the default latest node LTS image
+# Build stage
+FROM node:lts AS build
+
+## Create the solid community server directory
+RUN mkdir /community-server
```

Comment:
The WORKDIR on the next line implicitly runs mkdir. :D

Reply:
OMG! Great find, thanks for the tip :)